legal-api/src/legal_api/services/filings/validations/registration.py

```diff
@@ -83,6 +84,16 @@ def validate_business_type(filing: Dict, legal_type: str) -> list:
     return msg
 
 
+def validate_naics(filing: Dict, filing_type='registration') -> list:
+    """Validate naics."""
+    msg = []
+    naics_code_path = f'/filing/{filing_type}/business/naics/naicsCode'
+    if (naics_code := get_str(filing, naics_code_path)) and not NaicsService.find_by_code(naics_code):
```

Comment:
I think we should validation that the description in the filing matches up to the value pulled back from the NAICS endpoint too.

Reply:
ok will do that